core/types/l2trace.go

```diff
@@ -0,0 +1,139 @@
+package types
+
+import (
+	"io"
+	"sort"
+	"strings"
+
+	"github.com/scroll-tech/go-ethereum/rlp"
+)
+
```

Comment:
We need to create a `BlockResult` struct which contains a field of `[]*ExecutionResult`. Later, we need to add more data into this `BlockResult`.

Reply:
changed.